www/pages/docs/configuration.md

```diff
@@ -116,3 +116,23 @@ module.exports = {
   workspace: path.join(__dirname, 'www'),
 }
 ```
+
+### Markdown
```

Comment:
Minor, but can we keep the configuration sub nav alpha sorted?  (So this would go before _Meta_)

Reply:
yea thats fine